### PR TITLE
[DOCKER]: add docker-compose.yaml and Dockerfile

### PR DESCRIPTION
**Pull request by keda-vlad**

### CODE REVIEW

File: docker-compose.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    env_file: ./.env
+    ports:
+      - $MYSQL_LOCAL_PORT:$MYSQL_DOCKER_PORT
+    environment:
+      MYSQL_ROOT_PASSWORD: $MYSQL_ROOT_PASSWORD
+      MYSQL_USER: $MYSQL_USER
+      MYSQL_PASSWORD: $MYSQL_PASSWORD
+      MYSQL_DATABASE: $MYSQL_DB
+    healthcheck:
+      test: CMD-SHELL mysqladmin ping -u root -p$MYSQL_ROOT_PASSWORD
```
Comment (Someboty) [inline docker-compose.yaml:16]: Can we hide username too?

File: src/main/resources/application.properties [hunk 2]
```diff
@@ -1,6 +1,6 @@
 bot.name=${BOT_NAME}
 bot.token=${BOT_TOKEN}
-bot.chatId=${CHAT_ID}
+bot.chatId=387100384
```
Comment (Someboty) [inline src/main/resources/application.properties:3]: Can we hide it somewhere?

**Review by Someboty — COMMENTED** [review]
GJ! But we should do something about data security…

**Review by IhorKalaur — APPROVED** [review]

**Review by Denis-Balako — APPROVED** [review]
Awesome!

**Pull request merged**